public/app/core/components/Page/Page.tsx

```diff
@@ -50,7 +51,7 @@ export const Page: PageType = ({
   }, [navModel, pageNav, chrome, layout]);
 
   return (
-    <div className={cx(styles.wrapper, className)} {...otherProps}>
+    <div className={cx(styles.wrapper, className)} {...otherProps} data-testid={selectors.pages.Dashboard.wrapper}>
```

Comment:
@adela-almasan can you open a PR that removes this, this is the main Page component used by ALL pages in Grafana it should not have a test id of dashboard.wrapper

Reply:
https://github.com/grafana/grafana/pull/67402